src/main/osd/osd_warnings.c

```diff
@@ -165,33 +165,42 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
     if (osdWarnGetState(OSD_WARNING_RSSI) && (getRssiPercent() < osdConfig()->rssi_alarm)) {
         tfp_sprintf(warningText, "RSSI LOW");
         *displayAttr = DISPLAYPORT_ATTR_WARNING;
-        *blinking = true;;
+        *blinking = true;
         return;
     }
 #ifdef USE_RX_RSSI_DBM
     // rssi dbm
     if (osdWarnGetState(OSD_WARNING_RSSI_DBM) && (getRssiDbm() < osdConfig()->rssi_dbm_alarm)) {
         tfp_sprintf(warningText, "RSSI DBM");
         *displayAttr = DISPLAYPORT_ATTR_WARNING;
-        *blinking = true;;
+        *blinking = true;
         return;
     }
 #endif // USE_RX_RSSI_DBM
+#ifdef USE_RX_RSNR
+    // rsnr
+    if (osdWarnGetState(OSD_WARNING_RSNR) && (getRsnr() < osdConfig()->rsnr_alarm)) {
+        tfp_sprintf(warningText, "RSNR");
```

Comment:
Could be "RSNR LOW" or "RSNR DBM" as we have "RSSI LOW"

Reply:
done